thirdparty/ofqf/qoscserver.cpp

```diff
@@ -82,8 +82,8 @@ void QOscServer::readyRead()
         		      for ( ; i<size && data[ i ] != char( 0 ); ++i )
         		            path += data[ i ];
         
-        			    while ( data[ i ] != ',' ) ++i;
-        			    ++i;
+        			  while ( data[ i ] != ',' ) ++i;
+        			  ++i;
         		      while ( data[ i ] != char( 0 ) )
```

Comment:
Yeah, indenting of ofqf is a mess. Just this one looks like just a mistake and gcc says:
```
/home/bernhard/local-build/musescore-2.2/src/MuseScore/thirdparty/ofqf/qoscserver.cpp: In member function ‘void QOscServer::readyRead()’:
MuseScore/thirdparty/ofqf/qoscserver.cpp:82:17: warning: this ‘for’ clause does not guard... [-Wmisleading-indentation]
                 for ( ; i<size && data[ i ] != char( 0 ); ++i )
                 ^~~
MuseScore/thirdparty/ofqf/qoscserver.cpp:85:16: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘for’
                while ( data[ i ] != ',' ) ++i;
```